src/platform/EFR32/wifi/dhcp_client.cpp

```diff
@@ -0,0 +1,148 @@
+/*****************************************************************************
+ * @file
```

Comment:
If these files are to reside in the chip repo, I believe a license attribution/blurb update is required.

Reply:
Updated license attribution/blurb  for newly added files